husky_description/urdf/husky.urdf.xacro

```diff
@@ -37,6 +37,9 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
   <xacro:property name="husky_front_bumper_extend" value="$(optenv HUSKY_FRONT_BUMPER_EXTEND 0)" />
   <xacro:property name="husky_rear_bumper_extend" value="$(optenv HUSKY_REAR_BUMPER_EXTEND 0)" />
 
+  <!-- Height of the sensor arch in mm.  Must be either 510 or 300 -->
+  <xacro:arg name="sensor_arch_height"  default="$(optenv HUSKY_SENSOR_ARCH_HEIGHT 510)" />
```

Comment:
You mean something like HUSKY_SENSOR_ARCH_510 and HUSKY_SENSOR_ARCH_300?
My concern with that is that it could allow for a situation where someone's set both vars, which would result in two sensor bars being added.
Part of my thinking with having the height is that if we refactor the sensor_arch urdf we could allow for arbitrary heights in the future. Right now the only reason to limit it to 510 and 300 is because those are the filenames for the meshes that get imported.  But if we rebuild the arch's URDF to have boxes for the vertical segments (instead of a static mesh) and use a `height` parameter then we remove that limitation and have arches of any height.